CompareStringPermutation.java

```diff
@@ -0,0 +1,20 @@
+package com.katermar;
+
+import java.lang.reflect.Array;
+import java.util.Arrays;
+
+/**
+ * Created by USER on 5/15/2017.
+ */
+public class CompareStringPermutation {
+
+    public boolean compare(String first, String last) {
+        char[] a = first.toLowerCase().toCharArray();
+        char[] b = last.toLowerCase().toCharArray();
+
+        Arrays.sort(a);

```

Comment:
As discussed during the Hangout, time complexity of sort is O(NlogN) and there is a solution which works in O(N). Just wanted to point this out :)